EnvoyMobile.podspec

```diff
@@ -1,13 +1,17 @@
 Pod::Spec.new do |s|
-    s.name              = 'EnvoyMobile'
-    s.version           = '0.2.0'
-    s.author            = 'Lyft, Inc.'
-    s.summary           = 'Client networking libraries based on the Envoy project'
-    s.homepage          = 'https://envoy-mobile.github.io'
-    s.documentation_url = 'https://envoy-mobile.github.io/docs/envoy-mobile/latest/index.html'
-    s.social_media_url  = 'https://twitter.com/EnvoyProxy'
-    s.license           = { :type => 'Apache-2.0', :file => 'LICENSE' }
-    s.platform          = :ios
-    s.source            = { :http => "https://github.com/lyft/envoy-mobile/releases/download/v#{s.version}/envoy_ios_framework.zip" }
-    s.ios.vendored_frameworks = 'Envoy.framework'
+    s.name                = 'EnvoyMobile'
+    s.version             = '0.2.0'
+    s.author              = 'Lyft, Inc.'
+    s.summary             = 'Client networking libraries based on the Envoy project'
+    s.homepage            = 'https://envoy-mobile.github.io'
+    s.documentation_url   = 'https://envoy-mobile.github.io/docs/envoy-mobile/latest/index.html'
+    s.social_media_url    = 'https://twitter.com/EnvoyProxy'
+    s.license             = { :type => 'Apache-2.0', :file => 'envoy_ios_cocoapods/LICENSE' }
+    s.platform            = :ios, '10.0'
+    s.swift_version       = '5.1'
+    s.libraries           = 'resolv.9', 'c++'
+    s.frameworks          = 'SystemConfiguration'
+    s.source              = { :http => "https://github.com/lyft/envoy-mobile/releases/download/v#{s.version}/envoy_ios_cocoapods.zip" }
+    s.vendored_frameworks = 'envoy_ios_cocoapods/Envoy.framework'
+    s.source_files        = 'envoy_ios_cocoapods/Envoy.framework/Headers/*.h', 'envoy_ios_cocoapods/Envoy.framework/Swift/*.swift'
```

Comment:
You shouldn't need source files since you're vendoring a framework?

Reply:
Without this the framework fails `pod spec lint` because it's missing `libSwiftFoundation`, among other Swift libraries. I added some more explanations to the PR description - without this and manually adding a `.swift` file, the pod doesn't compile. Totally open to better ways to do this

Reply:
Is there a corresponding CocoaPods issue about this? Seems weird that no one else would have this case

Reply:
Not from what I could tell, but I'm happy to file one. Assuming it's only an issue for mixed Objective-C/Swift libraries?

Reply:
CP likely doesn't check that either way, but maybe you adding the `swift_version` was what broke it?

Reply:
Commenting out `swift_version` and `source_files` yields the following errors:

```
   Testing with `xcodebuild`. 
 -> EnvoyMobile (0.2.0)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | [iOS] xcodebuild:  note: Planning build
    - NOTE  | [iOS] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftFoundation'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftCompatibilityDynamicReplacements'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftDarwin'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftCoreFoundation'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftCore'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftCoreGraphics'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftObjectiveC'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftDispatch'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftSwiftOnoneSupport'
    - NOTE  | xcodebuild:  ld: warning: Could not find or use auto-linked library 'swiftCompatibility50'
    - NOTE  | xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Mind taking another look whenever you get a chance @keith?